tests/testthat/test-snapshot-palettes.R

```diff
@@ -25,11 +25,28 @@ test_that("nmfs_palette() fails", {
 })
 
 test_that("display_nmfs_palette() works", {
-  # Check that display_nmfs_palette returns a function
+  # Check that display_nmfs_palette returns an object
   urchin_palette <- display_nmfs_palette("urchin", 4)
-
-  expect_type(urchin_palette, "list")
-
+  
+  expect_type(urchin_palette, "object")
+  
   # Check that display_nmfs_palette() returns a snapshot
   expect_snapshot(urchin_palette$data)
 })
+
+test_that("nmfs_theme() works", {
```

Comment:
great idea to include these snapshot tests! I couldn't review the snaps for correctness as they were text files showing the structure of the plots (I think), so I am assuming that they have already been checked for correctness.